mod/state-transition/pkg/core/state_processor_genesis.go

```diff
@@ -106,17 +107,41 @@ func (sp *StateProcessor[
 		}
 	}
 
+	// Process deposits
 	for _, deposit := range deposits {
 		if err := sp.processDeposit(st, deposit); err != nil {
 			return nil, err
 		}
 	}
 
-	// TODO: process activations.
+	// Process activations
```

Comment:
solves a very important todo. Now:

- Effective balances are not changed every slot, when we perform `applyDeposit` but each epoch
- So genesis processing must explicitly activate validators